codegen/templates/http_client.tmpl

```diff
@@ -76,6 +79,37 @@ func {{$exportName}}(deps *module.Dependencies) Client {
 		deps.Default.Config.MustGetStruct("clients.{{$clientID}}.defaultHeaders", &defaultHeaders)
 	}
 
+	{{if ne $clientID "apprentice" -}}
+	apprenticeMethodNames := map[string]string{
+		"Apprentice::getRequest": "getRequest",
+		"Apprentice::getResponse": "getResponse",
+	}
+
+	apprenticeTimeout := time.Millisecond * time.Duration(
```

Comment:
Can you add an entry to the changelog that documents this? Now `clients.apprentice.timeout` and others are required. Or you can make them optional.

Reply:
i'll make it optional